pxr/usd/usdLux/shapingAPI.h

```diff
@@ -201,8 +229,29 @@ class UsdLuxShapingAPI : public UsdAPISchemaBase
     // --------------------------------------------------------------------- //
     // SHAPING:CONE:ANGLE 
     // --------------------------------------------------------------------- //
-    /// Angular limit off the primary axis to restrict the
-    /// light spread.
+    /// Angular limit off the primary axis to restrict the light
+    /// spread, in degrees.
```

Comment:
It looks like this has been rewritten since I last took a look at the docs/test images.  Do you know where this stands w.r.t. Renderman conformance at the moment?

Reply:
It's the same for coneAngle < 90 degrees.  Renderman seems to cap the value at 90 degrees - if coneAngle >= 90 degrees, it behaves as though comeAngle = 180 / coneAngle is disabled.